HW12/task_3.py

```diff
@@ -0,0 +1,18 @@
+from datetime import datetime
+
+
+def num_of_repeads(my_func):
+    print("How many times repeat func?")
+    times = input()
+    count = 1
+    while count <= int(times):
+        print(f"Function {my_func.__name__}\nRun on: {datetime.today()}")
+        count += 1
+
+
+@num_of_repeads
```

Comment:
There should be decorator with parameter like @num_of_repeads
```suggestion
@num_of_repeads(times=5)
```